package.json

```diff
@@ -101,7 +101,7 @@
     "replace": "1.2.1",
     "splashicon-generator": "0.2.12",
     "stylint": "1.5.9",
-    "tar": "4.4.18",
```

Comment:
quentin
[il y a 3 jours](https://mattermost.cozycloud.cc/cozycloud/pl/pjgp369agjdppj8t4gi1qssbkc)

Since https://github.com/cozy/cozy-drive/commit/09e89a4e32caef1e7058ddc53d36a3eb3f4f38b4#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485L120 this script is not used

Reply:
et vu que tu veux virer aussi tout ce qui est testcafé related, tu peux dégager tous les scripts genre create-deploy-test & delete-instance et hop vraiment plus de trace de tout ça :wink: